static-build/CMakeLists.txt

```diff
@@ -303,31 +303,31 @@ ExternalProject_Add(tarantool
 )
 
 enable_testing()
-ExternalProject_Get_Property(tarantool binary_dir)
-SET(TARANTOOL_BINARY_DIR ${binary_dir})
+ExternalProject_Get_Property(tarantool install_dir)
```

Comment:
does it mean one have to run `make install` to have an ability to run `make test`?

Reply:
static-build tests are run with the `ctest` command, not `make test`, see https://github.com/tarantool/tarantool/blob/master/static-build/README.md#usage

`ctest` requires `cmake` and `make` to be run.